### DISCUSSION TITLE
How do I run the software in debug mode?

### DISCUSSION THREAD

**Discussion by gleissonneves** [yuezk/GlobalProtect-openconnect#241] [General]
![Captura de tela de 2023-06-28 12-45-03](https://github.com/yuezk/GlobalProtect-openconnect/assets/53718407/0c27e372-bff9-431b-8574-bbdbaf54296e)

**Comment by yuezk** [comment]
The CLI has no debug mode.

**Reply by gleissonneves** [reply]
Okay, thank you very much
